recipes/guardian/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "guardian" %}
+{% set version = "1.3.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://git.ligo.org/cds/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5a6b76b842935e33cff0d123c6f71a994dfca3dc72994f9812afb12e218a8602
+
+build:
+  number: 0
+  skip: true  # [win]
```

Can this be `noarch` instead?
```suggestion
  noarch: python
```
(Windows doesn't need to pass)

---

@chrisburr, this isn't `noarch` appropriate, since it uses `scripts` in `setup.py`.

---

@duncanmmacleod Can you explain why the use of scripts makes this not noarch appropriate?